pkg/rtc/participant.go

```diff
@@ -2061,6 +2062,9 @@ func (p *ParticipantImpl) onAnyTransportNegotiationFailed() {
 		},
 	})
 	p.CloseSignalConnection()
+
+	// on a full reconnect, no need to supervise this participant anymore
+	p.supervisor.Stop()
```

Comment:
Do you think we can close the participant here?

Reply:
Yeah, in theory we should be able to, but the blast radius is higher, i. e. like closing peer connection on server side which might trigger other things on client side. Client should just leave this session and re-establish a new one, but just in case the leave message arrives later and other stuff via `close` trigger a different path on the client, that is behaviour untested. Even this change is unnecessary, but am trying to reduce `error` level logs that are not necessary.